Makefile

```diff
@@ -31,6 +31,13 @@ all build:
 	hack/build-go.sh $(WHAT)
 .PHONY: all build
 
+# Build Code with E2E tests artifact.
+# Example:
+# make e2e
+e2e:
+	hack/build-go.sh test/e2e/e2e.test
```

Comment:
Yes, the runner could benefit from improvements.  The e2e tests are in the
package order.

On Thu, Feb 18, 2016 at 6:13 PM, jay vyas <notifications@github.com> wrote:

> In Makefile
> <https://github.com/openshift/origin/pull/7049#discussion_r53398871>:
>
> > @@ -31,6 +31,13 @@ all build:
> >  	hack/build-go.sh $(WHAT)
> >  .PHONY: all build
> >
> > +# Build Code with E2E tests artifact.
> > +# Example:
> > +# make e2e
> > +e2e:
> > +	hack/build-go.sh test/e2e/e2e.test
>
> Hmmmmm
> https://github.com/openshift/origin/blob/master/test/extended/util/test.go
> Seems to be wrapping the existing tests with a customized runner that
> replaces e2e.go, by imementing the same functionality.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/7049/files#r53398871>.
>